apps/clock/style/alarm.css

```diff
@@ -75,6 +75,10 @@ ul.compact li label:active {
   display: none;
 }
 
+#time-select {
+  text-indent: 1000rem;
```

Comment:
are you sure that this is the best way to solve the issue? if so, please add a comment explaining why we need the text-indent (I know it's to move the caret away from the screen but we might forget it in the future and re-introduce the bug, specially since there is no way to write a test for it).

I assume the same problem is also present in all `<input type="time" />` (at least on the calendar app it does happen) so I would rather see this solved globally (eg. `shared/style/input_areas.css`)